src/app/components/ConnectorTypeCard/ConnectorTypeCard.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
Adding a new copy to prevent the cy test from failing, I will plan to update the old file, removing this one and also updating the cy tests.